changedetectionio/forms.py

```diff
@@ -229,6 +229,7 @@ class watchForm(commonSettingsForm):
 
     minutes_between_check = html5.IntegerField('Maximum time in minutes until recheck',
                                                [validators.Optional(), validators.NumberRange(min=1)])
+    use_seconds = BooleanField('Use seconds')
```

Comment:
better to just store things as seconds in the backend, but keep the current frontend/UI field, and migrate the data to a new field

Reply:
this field not required

Reply:
Still going to be working on this for a while.  Adding cron support with croniter, not terribly difficult.  UI will look like this:

![image](https://user-images.githubusercontent.com/16741693/150136507-5d1a3b94-48c3-436b-885d-6a69273e1d13.png)

With or without changing to seconds in the back end, will still need a field to display correct value and show the correct radio button on the edit page when editing an existing watch.  Data will still need to be migrated because I plan to change the value field name to time_between_check, so store.py will look like this:

![image](https://user-images.githubusercontent.com/16741693/150141005-896414bc-d86b-474b-8c23-390fd2c1b1dc.png)

The cron validator is simple with croniter - allows all valid syntax except @reboot for obvious reasons, and the init ticker logic remains simple with croniter calculating the next check after each trigger.

Reply:
The b42b86b push changes the minutes_between_check key, so it would need an upgrade or data migration feature.  I don't have a good handle on what that needs to consist of, so please comment about that.  Could also restore the minutes_between_check and include seconds as seconds_between_checks and avoid the data migration.  Thoughts on that too, please.

Reply:
Also haven't added queue size / status per your suggestion yet, but will before this is all sorted out.

Reply:
And of course I just noticed that I didn't include incrementing the next 'cron_next_check' after it triggers.  Will fix/push later today.

Reply:
Great catch!  I didn't realize croniter handled seconds.  Working on this now, and reverting back to minutes_between_check.

![image](https://user-images.githubusercontent.com/16741693/150652634-94535e43-29b1-40ff-8453-452cbedeb10d.png)

Reply:
Great catch! I didn't realize croniter handled seconds. Working on this now, and reverting back to minutes_between_check.  Also, I haven't addressed timezones, but should so that cron checks are stored and performed at client time instead of the server time where changedetection.io is running.

![image](https://user-images.githubusercontent.com/16741693/150652748-b0981158-d79c-4442-b1e0-9a5214c2ca4e.png)

Reply:
we can then translate "time between recheck" into a cron-tab type expression I think, sort of kills two birds with one stone :)

Reply:
Unfortunately it's not possible to convert all intervals into a single cron expression.  This has been explored a lot, and here's a stackoverflow post with 72 minutes as an example of the complications:  https://stackoverflow.com/questions/745901/how-to-do-a-cron-job-every-72-minutes.

However, all watches can be stored as a next_check epoch timestamp, so the ticker code can be simplified down to "if not watch['paused'] and (time.time() >= int(watch['next_check'])):", and after it triggers the next cron/minute timestamp is stored.  That approach should still maintain the minutes_before_check field for easy population in the edit form and settings form - otherwise the minutes needs to be deduced when those forms are called which adds more code than keeping the field intact.

I have that whole approach working, but I think it's essential to add history pruning in store.py that executes in __init__ after every ticker event so that short minute and second intervals don't blow up the json file - at 10 sec intervals it blows up fast!  For this I'm adding Unlimited (default) and 10/50/100 selectfield choices to global settings and individual watches like this:
![image](https://user-images.githubusercontent.com/16741693/150691043-3867db7e-f2b3-472f-b656-f84a0ffa8a3c.png)

I think all these features still belong in one PR because they're interconnected and dependent, so I'm going to keep working it here as "seconds" and we can give it a more comprehensive name later in a fresh PR.